hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocol/OzoneManagerProtocol.java

```diff
@@ -602,4 +604,12 @@ default long prepareOzoneManager(
       throws IOException {
     return -1;
   }
+
+  default PrepareStatusResponse getOzoneManagerPrepareStatus(long txnId)
```

Comment:
Please also document this javadoc for this method.

Reply:
Fixed.